include/ibamr/BrinkmanPenalizationStrategy.h

```diff
@@ -18,7 +18,7 @@
 
 /////////////////////////////// INCLUDES /////////////////////////////////////
 
-#include <ibamr/config.h>
+#include "ibtk/ibtk_utilities.h"
```

Comment:
We need to update the CI to check this, but every one of our headers needs to include either `ibtk/config.h` or `ibamr/config.h`. Can you add it back in?

Reply:
Added that.